MovingBlocks/Terasology
# Update CZ translation

### Contains
Czech translation updates:
- changed "enter-username-message" string to comply with changes as stated in #2921
- added translations for new strings (identity and telemetry mostly)
- fixed typos in few translations
- a little bit of reordering to be same as EN lang for better comparison

### How to test
1. Run the game
2. Switch to the Czech language
3. See everything translated :)

### Outstanding before merging
- nothing
